core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -20,9 +20,9 @@ <h4 class="bold-title">
                     <p class="step-title">Choose the type of service you want to create<span>*</span></p>
                 </div>
                 <div class="space-between content">
-                    <section class="pad-left0 service-box-section">
-                        <div tabindex="0" class="service-box" [ngClass]="{'service-box-selected': typeOfService == 'api'}" (keyup.enter)="changeServiceType('api')" (click)="changeServiceType('api')">
-                            <div class="icon-icon-api selected"></div>
+                    <section class="pad-left0 service-box-section disabled-box">
```

Comment:
This should be disabled only when `provider = azure`.

Reply:
Done